locales/base/translation.json

```diff
@@ -2092,6 +2092,8 @@
   "viewOnArbiscan": "View on Arbiscan",
   "viewOnOPMainnetExplorer": "View on OP Mainnet Explorer",
   "viewOnOPSepoliaExplorer": "View on OP Sepolia Explorer",
+  "viewOnPolygonPoSScan": "View on PolygonPoSScan",
```

Comment:
could value just be "View on PolygonScan"? `PolygonPoSScan` doesn't seem very readable for the user